velox/functions/prestosql/Reduce.cpp

```diff
@@ -155,8 +163,10 @@ class ReduceFunction : public exec::VectorFunction {
     }
 
     // Apply output function.
-    VectorPtr localResult;
-    auto outputFuncIt = args[3]->asUnchecked<FunctionVector>()->iterator(&rows);
+    VectorPtr localResult =
```

Comment:
no need to create a new basevector here, expression eval will create an appropriate one if required